build-service.py

```diff
@@ -216,17 +247,38 @@ def build_request():
         app.run(host="0.0.0.0", port=args["--port"], debug=args["--verbose"])
         exit(0)
 
-    # image_name = build_repo(args["<repo-url>"], args["--branch"], args['--deploy-conf'], args['--service-conf'])
-
-    try:
-        image_name = build_repo(
-            args["<repo-url>"],
-            args["--branch"],
-            args["--deploy-conf"],
-            args["--service-conf"],
-        )
-    except Exception as e:
-        logging.error(f"Error building repo: {e}")
-        exit(1)
-
-    print(f"Repository built successfully as '{image_name}'")
+    with tempfile.TemporaryDirectory(prefix="kaas-repo-build-") as repo_dir:
+        repo = args["<repo-url>"]
+        branch = args["--branch"]
+        clone_repo(repo, branch, repo_dir)
+        deploy_conf = get_deploy_conf(args["--deploy-conf"], repo_dir)
+        service_conf = get_service_conf(args["--service-conf"], repo_dir)
+
+        # load kube config from ~/.kube/config
+        kubernetes.config.load_kube_config()
+        kubeneretes_api = kubernetes.client.AppsV1Api()
+        try:
+            if args["--restart"]:
+                image_name = args["--restart"]
+                kubeneretes_api.patch_namespaced_deployment(
+                    name=image_name, namespace="default", body=deploy_conf
+                )
+                print(f"Successfully restarted '{image_name}'")
+            elif args["--delete"]:
+                image_name = args["--delete"]
+                kubeneretes_api.delete_namespaced_deployment(
+                    name=image_name, namespace="default"
+                )
+                print(f"Successfully deleted '{image_name}'")
```

Comment:
Restart & delete should also be exposed via the daemon as well so that it can be controlled via the web dashboard, e.g. via `PATCH` and `DELETE` requests  to `/build` or on different endpoints.